docs/system/k8s-metrics.md

```diff
@@ -743,5 +748,113 @@ This metric is [recommended][MetricRecommended].
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Daemonset Metrics
```

Comment:
It would be nice to be consistent with capitalization of `Daemonset` vs `DaemonSet` everywhere. K8s documentation uses DaemonSet, so that seems preferred

Reply:
Good point. Fixed.